src/dao/ERC20DAO.js

```diff
@@ -26,9 +26,6 @@ export default class ERC20DAO extends AbstractTokenDAO {
   }
 
   getSymbol () {
-    if (!this._symbol) {
-      throw new Error('symbol is undefined')
```

Comment:
symbol is obligatory, so this code obligatory as well
TokenForm async validation tries to receive symbol through the Contract.symbol.call(), if there is no symbol getter function or symbol public variable, then such call will through error, that will be caught. Then user should provide value for symbol manually.